autoPyTorch/data/abstract_data_manager.py

```diff
@@ -0,0 +1,34 @@
+import abc
+from typing import Any, Dict, List
+
+import numpy as np
+
+
+class AbstractDataManager():
```

Comment:
Maybe we could add some documentation here?

Reply:
This was a WA because we didn't have the dataset class! I am going to rebase and remove this which is no longer needed